scif_app_recipes/opendronemap_v0.9.1_ubuntu16.04.scif

```diff
@@ -0,0 +1,23 @@
+%appinstall odm
+
+%apprun odm
+    docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v scif_input:/scif_input/images -v scif_output:/scif_output --user root agdrone/transformer-opendronemap:2.2 "${@}"
```

Comment:
This will break on HPC.

Reply:
More specifically, you can't run Docker on most HPC systems. You have to use Singularity (95%), or one of the other 'safe' containerization solutions.

If you want to use this mechanism of a SCIF app running a container I suggest you (@Chris-Schnaufer ) test this on UA HPC and Bridges.